frontend/src/App.tsx

```diff
@@ -177,11 +173,11 @@ export default function App() {
     // The name is not saved in a cookie until a game is started.
   };
 
-  const handleEloId = (eloId: string) => {
+  const handleIdToken = (idToken: string) => {
     updateState((draftState) => {
-      draftState.eloId = eloId.slice(0, maxEloIdLen);
+      draftState.idToken = idToken;
     });
-    // The eloId is not saved in a cookie until a game is started.
+    // idToken not saved until player plays their first game
```

Comment:
Maybe you can just delete this comment, since the idToken is not saved in a cookie.

Otherwise, expand on it:
1. what does it mean for an idToken to be saved?
2. where is it saved? in our database? in auth0's backend?
3. why is it not saved until a player plays their first game?